packages/lesswrong/lib/collections/users/collection.ts

```diff
@@ -15,6 +15,7 @@ export const Users: ExtendedUsersCollection = createCollection({
   collectionName: 'Users',
   typeName: 'User',
   schema,
+  // FIXME: switching this to postgres will cause `insertHashedLoginToken` to fail at the moment because $addToSet hasn't yet been implemented
```

Comment:
@oetherington you're probably already aware of this but I've added this warning just in case, this is the offending function:

```
async function insertHashedLoginToken(userId: string, hashedToken: string) {
  const tokenWithMetadata = {
    when: new Date(),
    hashedToken
  }

  await Users.rawUpdateOne({_id: userId}, {
    $addToSet: {
      "services.resume.loginTokens": tokenWithMetadata
    }
  });
};
```